orchestration/dagster_orchestration/hca_orchestration/solids/load_hca/load_data_files.py

```diff
@@ -1,12 +1,230 @@
-from dagster import solid, Nothing, InputDefinition
+from typing import Iterator
+
+from dagster import composite_solid, solid, Nothing, InputDefinition, Int, configured, Failure
 from dagster.core.execution.context.compute import AbstractComputeExecutionContext
+from dagster.experimental import DynamicOutput, DynamicOutputDefinition
+from data_repo_client import ApiException
+from data_repo_client import JobModel
+from google.cloud import bigquery
+from google.cloud.bigquery.client import RowIterator
+
+from hca_manage.manage import JobId
+from hca_orchestration.contrib.bigquery import build_query_job_using_external_schema, build_extract_job
+from hca_orchestration.contrib.retry import is_truthy, retry
+from hca_orchestration.resources.config.hca_dataset import TargetHcaDataset
+from hca_orchestration.resources.config.scratch import ScratchConfig
+from hca_orchestration.solids.data_repo import wait_for_job_completion
+from hca_orchestration.support.typing import HcaScratchDatasetName
+from dagster_utils.typing import DagsterConfigDict
+from dagster_utils.contrib.google import gs_path_from_bucket_prefix
+
+
+FILE_LOAD_TABLE_BQ_SCHEMA = [
+    {
+        'mode': 'NULLABLE',
+        'name': 'source_path',
+        'type': 'STRING'
+    },
+    {
+        'mode': 'NULLABLE',
+        'name': 'target_path',
+        'type': 'STRING',
+    }
+]
+
+
+@solid(
+    required_resource_keys={"bigquery_client", "scratch_config", "storage_client", "target_hca_dataset"},
+    input_defs=[InputDefinition("scratch_dataset_name", HcaScratchDatasetName)],
```

Comment:
wanted to confirm again whether the `input_def` here is required - i believe dagster should be able to infer it from the argument and annotation